contribs/docker/wazo-confd-mock/mock-wazo-confd.py

```diff
@@ -20,6 +20,12 @@
     'conferences': {},
     'contexts': {},
     'infos': {},
+    'ingresses': {
+        'items': {
+            'uri': 'localhost',
+            'uuid': '60f123e6-147b-487c-b08a-36395d43346e',
+        },
+    },
```

Comment:
Avoid to add default resource in the container. With time, they are really painful to maintain across all repos.
Example how to inject data for your test: https://github.com/wazo-platform/wazo-setupd/blob/master/integration_tests/suite/helpers/confd.py